Add two sections to dryville_story.md

wildbio opened the pull request:
### I added two sections to dryville_story.md (see #8):
1. Getting Water to Your Homes
2. Dryville's First Water Works

wildbio committed:
add getting-water-to-your-homes section

wildbio committed:
add dryvilles-first-water-works-section

jesse-ross reviewed (changes requested):
This looks good! There is only one change I would request. In the Getting Water to Your Homes section, there should be a link for "water does flow downhill". Could you please add that?

wildbio committed:
Add links

In the "Getting Water to Your Homes" section, I added two links: one for "Water does flow downhill" and one for "creek".

jesse-ross reviewed (approved):
Oh, great, you caught another link, too! Looks good, feel free to merge 😻

wildbio commented on dryville_story.md:
```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
+
+Since you appointed yourself mayor Horace of Dryville, you naturally chose the best spot for your home — on top of the hill. Great view, but it sure is a pain lugging pails of water (at [over 8 pounds a gallon](https://www.usgs.gov/special-topics/water-science-school/science/water-density), your 100 gallons of water per day gets heavy in a hurry!) from the creek up the hill all day long. And now  ...
+
+The problem is, the houses are higher than the creek, and water does not flow uphill. [Water does flow downhill](https://www.usgs.gov/special-topic/water-science-school/science/watersheds-and-drainage-basins), so you build a big water storage tank on a hill (hopefully at the highest point in town) and establish a water brigade to fill it full of water. You can run pipes from the storage tank down ...
```
Link added here

wildbio commented on dryville_story.md:
```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
+
+The problem is, the houses are higher than the creek, and water does not flow uphill. [Water does flow downhill](https://www.usgs.gov/special-topic/water-science-school/science/watersheds-and-drainage-basins), so you build a big water storage tank on a hill (hopefully at the highest point in town) and establish a water brigade to fill it full of water. You can run pipes from the storage tank down ...
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the [creek](https://www.usgs.gov/special-topic/water-science-school/science/rivers-streams-and-creeks) uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outsid ...
```
Link added here

wildbio commented:
Thanks, Jesse! I'm getting the message that there are no conflicts between this branch and the base branch, but, "Only those with write access to this repository can merge pull requests."

jesse-ross commented:
Right! Merging now.
